Translation/en/Text/Main/ActionScene.txt

```diff
@@ -90,8 +103,8 @@ r:^※コイカツポイントを(\d+)pt消費します。$=※ Consumes $1 Koik
 お泊り=Staying
 お祈り=Prayer
 所持数=Num Items
-滞在先=Accommodations
-現在地=Location
+//滞在先=Bookings
+//現在地=Place
 恋人数=Num Lovers
```

Comment:
This seems weird, maybe change to just Lovers or Lover count?

Reply:
Hmm, I didn't make this. I'll see ingame what was GeBo's intention on translating this.